src/Google/Client.php

```diff
@@ -795,7 +796,7 @@ public function execute(RequestInterface $request, $expectedClass = null)
     // this is where most of the grunt work is done
     $http = $this->authorize();
 
-    return Google_Http_REST::execute($http, $request, $expectedClass, $this->config['retry']);
+    return Google_Http_REST::execute($http, $request, $expectedClass, $this->config['retry'], $this->config['retry_map']);
```

Comment:
Confirmed this does the trick.

LGTM, except for one tiny nit. This is now breaking the line length guidelines (my fault, it was the way I formatted the code in my suggestion). Could we break this apart over some newlines?

```php
return Google_Http_REST::execute(
    $http,
    $request,
    $expectedClass,
    $this->config['retry'],
    $this->config['retryMap']
);
```

Reply:
Done